app/views/catalog/_home_text.html.erb

```diff
@@ -1,49 +1,41 @@
 <div class="row">
   <div class="col-sm-12">
-       <%- if params.has_key?(:lib) %>
-      <%- @library = params[:lib] %>
+    <%- if params.has_key?(:lib) %>
+      <%- @library = $brand %>
```

Comment:
Are the changes below this point just changing the spacing of the file?

Reply:
Yes - sorry about that!